editor.planx.uk/src/@planx/components/shared/Preview/SimpleExpand.tsx

```diff
@@ -33,7 +33,8 @@ const SimpleExpand = ({ children, buttonText }: any) => {
         <Button
           className={classes.button}
           onClick={() => setShow(!show)}
-          // Needs aria-controls, aria-expanded
+          aria-expanded={show}
+          // Needs aria-controls
```

Comment:
`aria-controls` is still a bit tricky here and I haven't worked out exactly what it is controlling because we pass general `{children}` here? I think it's easy to come back to and doesn't need to hold up initial merge though?